SPECS/grpc/grpc.spec

```diff
@@ -101,7 +139,16 @@ find %{buildroot} -name '*.cmake' -delete
 %license LICENSE
 %{_bindir}/grpc_*_plugin
 
+#imported from fedora
```

Comment:
Please just mention that we've taken the spec changes from Fedora X (where "X" is the version number) inside your changelog entry. We should also mention their license, which is MIT for Fedora specs.

Reply:
added comment in my change log